prov/rxd/src/rxd_msg.c

```diff
@@ -343,13 +343,10 @@ static struct rxd_x_entry *rxd_tx_entry_init_msg(struct rxd_ep *ep, fi_addr_t ad
 		tx_entry->num_segs = 1;
 	}
 
-	tx_entry->bytes_done = rxd_init_msg(&ptr, tx_entry->iov,
-					    tx_entry->iov_count,
-					    tx_entry->cq_entry.len,
-					    max_inline);
+	tx_entry->bytes_done = rxd_init_msg(&ptr, tx_entry->iov,tx_entry->iov_count,
```

Comment:
Nit: this change makes the line exceed 80 characters and also is missing a space between arguments

Reply:
@aingerson  Ok. Done!